web/content/_index.md

```diff
@@ -6,7 +6,13 @@ This is work-in-progress documentation site for <a href="https://www.theforeman.
 
 Official documentation [is available here](https://theforeman.org/manuals/nightly/index.html).
 
-Available documentation sets are:
+Available upstream documentation sets are:
 
 * [Nightly](/nightly.html)
 * [2.3](/2.3.html)
+
+Other relevant documentation:
+
+* [Red Hat Satellite](https://access.redhat.com/documentation/en-us/red_hat_satellite) is a distribution of Foreman and several plugins including Katello, Remote Execution, Discovery, Bootdisk, OpenSCAP for the enterprise.
+* [ATIX orcharhino](https://docs.orcharhino.com/or/docs/index.html) is a distribution of Foreman and many plugins providing vendor-independent management of Red Hat, Suse, Oracle and Debian/Ubuntu nodes
+* [Red Hat Satellite Performance Tuning](https://access.redhat.com/solutions/1257143) is a set of performance-related resources, a free Red Hat portal account is required to access the content. Note that while some of the guidance here might help you, Red Hat Satellite is a combination of Foreman and Katello. The latest Satellite release is always a few versions behind the latest Foreman release
```

Comment:
```suggestion
* [Red Hat Satellite Performance Tuning](https://access.redhat.com/solutions/1257143) is a set of performance-related resources, a free Red Hat portal account is required to access the content. Note that while some of the guidance here might help you, Red Hat Satellite is a combination of Foreman and Katello. 

The latest Satellite and orcharhino releases are always behind the latest Foreman release.
The downstream documentation is meant as an extension, not a substitution for first party Foreman documentation.
```

Reply:
I think that the motivation of the last sentence was to tell the reader that Performance Tuning might be out of date.

You are changing this to a more generic statement about the obvious lag of downstream and upstream. I am not sure if that's needed? @melcorr

Reply:
For me, this makes the most sense. My logic:
orcharhino & Satellite != Foreman nightly (same for the Performance Tuning Guide) > a generic warning would be neat.